charts/cosmo-dashboard/values.yaml

```diff
@@ -79,6 +79,10 @@ cert:
 
 insecure: false
 
+ldapURL: ""
+ldapUserAttr: ""
+ldapBaseDn: ""
+
```

Comment:
It seems preferred like this.
(Please add setting example)

```yaml
auth:
  ldap:
    enabled: false
    # url is ldap server url. protocol is supported "ldap://" or "ldaps://"
    url: "ldap://example.com"
    # baseDn is xxxxxx
    baseDn: "example.example"
    tls:
      insecure: false
      secretName: ""
```

Reply:
ok.
Do you want to move the insecure flag and others under auth?